app/views/home/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="home-buttons"><%= link_to markets_path, class: 'btn btn-success btn-lg' do %>
+Market
+<% end %>
+
+<%= link_to vendors_path, class: 'btn btn-success btn-lg' do %>
+Vendor
+<% end %>
+
+<div class="welcome"></div>
+</div>
```

Comment:
How are you rendering the photo that shows on your homepage? It's a little off-center. The buttons you have on this view are also not always centered, depending on screen size.